src/common/locations.ts

```diff
@@ -148,3 +148,11 @@ export function getAdjacentCounties(fips: string): string[] {
   assert(fips in ADJACENT_COUNTIES, `${fips} not found in adjacency list.`);
   return ADJACENT_COUNTIES[fips].adjacent_counties;
 }
+
+export function getCounty(stateId: string, fullFips: string) {
```

Comment:
If we have the full fips, why don't we just use findCountyByFips()?

https://github.com/covid-projections/covid-projections/pull/1329/files#diff-4dab089aa9f4d6461076b7f21ec50a8fR69